gym/envs/registration.py

```diff
@@ -456,10 +456,13 @@ def make(self, path: str, **kwargs) -> Env:
             logger.info("Making new env: %s (%s)", path, kwargs)
         else:
             logger.info("Making new env: %s", path)
-        spec = self.spec(path)
```

Comment:
This could have failed before if you requested the unversioned environment. `spec()` would try to lookup that version and immediately fail.